internal/pkg/cli/svc_logs.go

```diff
@@ -193,6 +217,23 @@ func (o *svcLogsOpts) Execute() error {
 	return nil
 }
 
+func (o *svcLogsOpts) setPreviousStoppedTaskID() (string, error) {
+	svcDesc, err := o.newSvcDescriber().DescribeService(o.appName, o.envName, o.name)
+	if err != nil {
+		return "", fmt.Errorf("describe serivce %s: %w", o.name, err)
+	}
+	if len(svcDesc.StoppedTasks) > 0 {
+		taskID, err := awsecs.TaskID(aws.StringValue(svcDesc.StoppedTasks[0].TaskArn))
```

Comment:
Did we validate if the tasks returned are sorted by timestamp? if so can we add a comment here highlighting why it's okay to grab the first item

Reply:
I read through AWS SDK documentation, and nowhere they mentioned that they sort the list of tasks by the stopped time but for me I was getting it in sorted manner consistently but after you mentioned this I stopped almost 10 tasks and then checked and sometimes it is not sorted so for safer side I have added sorting logic based on stopped time. Now it will be sorted no matter how we get it from `listTasks` api